bin/debug/intake_single_user.py

```diff
@@ -30,4 +30,4 @@
     else:
         sel_uuid = ecwu.User.fromEmail(args.user_email).uuid
 
-    epi.run_intake_pipeline("single", [sel_uuid])
+    epi.run_intake_pipeline("single", [sel_uuid])
```

Comment:
what is the change here?

Reply:
How exactly do you I revert back to the old version of these files?

Reply:
if these changes are all in a few commits, you can use `git revert`. Otherwise, you can clone a new repo and copy over the original files

https://stackoverflow.com/a/6971775/4040267